public/app/plugins/datasource/cloudwatch/components/LogsQueryField.tsx

```diff
@@ -143,9 +142,31 @@ export class CloudWatchLogsQueryField extends React.PureComponent<CloudWatchLogs
 
   onChangeQuery = (value: string, override?: boolean) => {
     // Send text change to parent
-    const { query, onChange, onRunQuery } = this.props;
+    const { query, onChange, onRunQuery, datasource, exploreMode } = this.props;
     const { selectedLogGroups, selectedRegion } = this.state;
 
+    // TEMP: Remove when logs/metrics unification is complete
```

Comment:
Seems like this will conflict with https://github.com/grafana/grafana/pull/24392 that I am trying to merge at this moment. Should we rather use this solution? Is it related to the group by solution?